packages/xo-web/src/xo-app/home/index.js

```diff
@@ -5,6 +5,7 @@ import ActionButton from 'action-button'
 import Button from 'button'
 import CenterPanel from 'center-panel'
 import classNames from 'classnames'
+import cookies from 'cookies-js'
```

Comment:
Is it sorted?

Reply:
It's my fault. it's not sorted. 
To avoid unrelated changes, I will import it after `Component`.